extras/rya.pcj.fluo/pcj.fluo.app/src/main/java/org/apache/rya/indexing/pcj/fluo/app/IncrementalUpdateConstants.java

```diff
@@ -30,15 +30,18 @@
     public static final String TYPE_DELIM = "<<~>>";
 
     //to be used in construction of id for each node
-    public static final String SP_PREFIX = "STATEMENT_PATTERN";
-    public static final String JOIN_PREFIX = "JOIN";
-    public static final String FILTER_PREFIX = "FILTER";
-    public static final String AGGREGATION_PREFIX = "AGGREGATION";
-    public static final String QUERY_PREFIX = "QUERY";
-    public static final String PROJECTION_PREFIX = "PROJECTION";
-    public static final String CONSTRUCT_PREFIX = "CONSTRUCT";
-    public static final String PERIODIC_QUERY_PREFIX = "PERIODIC_QUERY";
-    
+    public static final String TRIPLE_PREFIX = "T";
```

Comment:
No.  Just more efficient.  Since I was making so many changes to how results were stored, I figured it was extremely unnecessary to have such large prefixes.  Also, when specifying splits in the fluo app, it is necessary to indicate the prefixes that you want to split over.  This becomes cumbersome with such large prefixes.